src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -289,6 +290,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
                 case MessageID.IDS_DefaultInterfaceImplementation: // semantic check
                 case MessageID.IDS_BaseTypeInBaseExpression:
                 case MessageID.IDS_OverrideWithConstraints: // semantic check
+                case MessageID.IDS_FeatureAllowGenericAttributes: // semantic check, PROTOTYPE(generic-attributes) need to attach feature to the proper language version once we have one
```

Comment:
The feature can be attached to LangVer "Preview" (not sure if it's available in this feature branch, or the feature branch needs to be refreshed first).

Reply:
done by you